packages/protocol/src/data/inscription.ts

```diff
@@ -721,10 +720,6 @@ export interface OutlineNode {
   title: string;
   type: string;
 }
-export interface ProcessMetaRequest {
-  context: InscriptionContext;
-  processId: string;
-}
```

Comment:
not used?

Reply:
I think this one is from the outline...

Reply:
Possibly, but I don't know for sure. I simply generated the latest inscription.ts from my core branch and replaced it.